profile/profile.json

```diff
@@ -0,0 +1,133 @@
+{
+  "education": [
+    {
+      "awards_achievements": [
+        "GPA: 3.91/4.0"
+      ],
+      "degree": "Cert.Univ. full time Computer Science and Information Technology",
+      "department": "Computer Science",
+      "duration": "2022-2023",
+      "school_name": "McGill scs"
+    },
+    {
+      "awards_achievements": [
+        "Final grade 110/110"
+      ],
+      "degree": "Master of Architectural Engineering",
+      "department": "Architecture",
+      "duration": "2004-2010",
+      "school_name": "Polytecnic of Turin (Italy)"
+    }
+  ],
+  "github_repo_root_url": "https://github.com/jarviscanada/jarvis_data_eng_alfettox",
+  "highlighted_projects": [
+    {
+      "description": "This application realized with Android Studio provide a graphical interface to access to Yelp API and allows the user to search any kind of commerce in a specific town.",
+      "git_url": "",
+      "name": "Android yelp clone"
+    },
+    {
+      "description": "Realized a complete website for a restaurant in montreal with home page, menus, gaming page, contacts. The website has been created using HTML, CSS, Javascript.",
+      "git_url": null,
+      "name": "Restaurant Website"
+    }
+  ],
+  "jarvis_projects": [
+    {
+      "description": "Designed and implemented a scalable Linux cluster monitoring agent using bash scripts. The system allows users to keep track of a cluster's hardware and usage information in real-time. The data is updated regularly using the cron command, ensuring that the information about the cluster's performance is always up-to-date. Used languages are bash scripting, sql scripting, markdown.  Used technologies are Linux, bash, docker, PostgreSQL, IntelliJ, draw.io",
+      "git_url": "/linux_sql",
+      "name": "Cluster Monitor"
+    },
+    {
+      "description": [
+        "Twitter App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam.",
```

Comment:
Dont forget to remove the sample text! Once you complete the project you can add the section back in